public_html/new_html/src/WikiText/WikiParse/src/ParserTemplate.php

```
@@ -0,0 +1,61 @@
+            $this->name = $params[0];
+            for ($i = 1; $i < count($params); $i++) {
+                $param = $params[$i];
+                if (strpos($param, "=") !== false) {
+                    $parts = explode("=", $param, 2);
+                    $key = trim($parts[0]);
+                    $value = trim($parts[1]);
+                    $data[$key] = $value;
+                } else {
+                    $data[$i] = $param;
+                }
+            }
```

PHPMD (2.15.0)

Avoid using WikiConnect\ParseWiki\count() function in for loops. (Design Rules)

(CountInLoopExpression)